cms/envs/common.py

```diff
@@ -1264,7 +1264,7 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-HELP_TOKENS_INI_FILE = REPO_ROOT / "docs" / "cms_config.ini"
+HELP_TOKENS_INI_FILE = REPO_ROOT / "cms" / "envs" / "help_tokens.ini"
```

Comment:
@andy-armstrong This seems to be working on the sandbox.  Is there a reason why you are concerned about devops need to review?